nautobot_chatops/integrations/grafana/diffsync/sync.py

```diff
@@ -22,6 +28,9 @@ def run_dashboard_sync(overwrite: bool = False) -> Union[str, None]:
     Args:
         overwrite (bool): Overwrite Nautobot data and delete records that are no longer in Grafana.
     """
+    if not DIFFSYNC_INSTALLED:
+        raise ImportError("DiffSync is not installed. Please install DiffSync to use the grafana integration.")
```

Comment:
ImportError for these calls feels weird to me - shouldn't it be something like RuntimeError?

Reply:
I was going to use RuntimeError but then I decided to replicate the exception you would get if you tried to import diffsync in this function. It seemed more appropriate to use ImportError here but I'm happy to change it to RuntimeError if you think that's better here.

Reply:
Would be my preference, but I don't feel strongly about it.

Might be good either way to add a `Raises:` section to the docstring documenting this behavior?